### PR TITLE
filter movie multi condition

### PR DESCRIPTION
**Pull request by namnn17b3**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/74039)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image
image
image
image

### CODE REVIEW

**Comment by namnn17b3** [comment]
ready

File: src/dtos/req/movie/moive.all.req.dto.ts
Comment (huongnt-2545) [inline src/dtos/req/movie/moive.all.req.dto.ts:1]: tên file -> `movie.all....ts`

**Comment by namnn17b3** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**